src/main/java/mike/main.java

```diff
@@ -1,12 +1,8 @@
-package mike;
-
-import java.io.IOException;
 import mike.cli.CommandLine;
 
 public class main {
```

Comment:
Need "package mike;" up top or Gradle won't find the main class and won't be able to build/run.

Reply:
Okay. Probably got removed when moving stuff around